cas-management-webapp-support/build.gradle

```diff
@@ -4,9 +4,12 @@ dependencies {
     compile(group: 'org.pac4j', name: 'spring-webmvc-pac4j', version: springWebmvcPac4jVersion) {
         exclude(module: 'spring-webmvc')
         exclude(module: 'spring-core')
+        exclude(module: 'slf4j-api')
         exclude(module: 'pac4j-core')
     }
-    compile group: 'org.pac4j', name: 'pac4j-core', version: pac4jVersion
+    compile(group: 'org.pac4j', name: 'pac4j-core', version: pac4jVersion) {
```

Comment:
Yes. Apparently, it's possible: https://stackoverflow.com/questions/9547170/in-gradle-how-do-i-declare-common-dependencies-in-a-single-place

I'll handle this in a separate pull.